README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Microverse-Project1
 
-> One paragraph statement about the project.
+> Project to be familiarized with Linters and Gitflow. 
 
-![screenshot](./app_screenshot.png)
+![screenshot](./template-screenshot.PNG)
 
-Additional description about the project and its features.
+In this repo no complex coding is required, our goal here is to master all of the tools and best practices we learn about setup a linter for Html and CSS file and Git/Github.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML/CSS
+- Technologies used: Linter/Git/Github
 
 ## Live Demo
```

Comment:
- [ ] **[OPTIONAL]** The demo link is broken. You can set up the live link like this:
- Go to this link (https://github.com/RolandM99/Setup-linter/settings/pages).
- Select the branch name
- Select root
- Click the Save button

Your project will be deployed by Github Pages